operator/v1/types_ingress.go

```diff
@@ -1346,6 +1385,59 @@ type IngressControllerHTTPHeaders struct {
 	// +nullable
 	// +optional
 	HeaderNameCaseAdjustments []IngressControllerHTTPHeaderNameCaseAdjustment `json:"headerNameCaseAdjustments,omitempty"`
+
+	// headersManipulation specifies options for set/replace/append/delete headers.
+	HeadersManipulation IngressControllerHeadersManipulation `json:"headersManipulation,omitempty"`
```

Comment:
Not sure why there is a need of having a `Response` field in the `IngressControllerHeadersManipulation`. 
Cant we user directly `[]IngressControllerSetReplaceAppendDeleteHTTPHeaders`
```suggestion
	HeadersManipulation []IngressControllerSetReplaceAppendDeleteHTTPHeaders`json:"headersManipulation,omitempty"`
```